scaling_performance/managing_hugepages.adoc

```diff
@@ -0,0 +1,87 @@
+[[scaling-performance-managing-huge-pages]]
+= Managing Huge Pages
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+
+== Introduction
+
+[IMPORTANT]
+====
+Managing huge pages is a Technology Preview feature only.
+ifdef::openshift-enterprise[]
+Technology Preview features are not supported with Red Hat production service
+level agreements (SLAs), might not be functionally complete, and Red Hat does
+not recommend to use them for production. These features provide early access to
+upcoming product features, enabling customers to test functionality and provide
+feedback during the development process.
+
+For more information on Red Hat Technology Preview features support scope, see
+https://access.redhat.com/support/offerings/techpreview/.
+endif::[]
+====
+
+In {product-title}, applications in a pod can allocate and consume pre-allocated
+huge pages. This topic describes how.
+
+[[huge-pages-prerequisites]]
+== Prerequisites
+
+. Nodes must pre-allocate huge pages in order for the node to report its huge
+ page capacity. A node can only pre-allocate huge pages for a single size.
+
+. `HugePages` must be set to `true` across the system:
++
+----
+--feature-gates=HugePages=true
```

Comment:
This is the kubelet argument format. but in openshift we have the yaml-based kubelet config. please refer back to the cpu manager docs for an example of how to set a feature gate in the node-config.yaml.  Then replace this line with the yaml example.